mne/decoding/search_light.py

```diff
@@ -316,7 +327,7 @@ def _sl_init_pred(y_pred, X):
     return y_pred
 
 
-def _sl_score(estimators, X, y):
+def _sl_score(estimators, scoring, X, y):
     """Aux. function to score SearchLight in parallel.
 
```

Comment:
scoring is missing in the doc: You can add there the lines I suggested above to explicit what is going on.

Formally, I think it's not perfect because that means the functions does very different things depending on the input (i.e. X becomes a y_pred), but I guess it's private so we'll deal with it.